api-specificatie/zrc/1.1.x/openapi.yaml

```diff
@@ -134,6 +139,11 @@ paths:
                 type: string
               description: 'Geeft een specifieke API-versie aan in de context van
                 een specifieke aanroep. Voorbeeld: 1.2.1.'
+            Warning:
+              schema:
+                type: string
+              description: Geeft een endpoint-specifieke waarschuwing, zoals het uitfaseren
```

Comment:
Is dit de inhoud die de warning teruggeeft? Zo ja, dan moet deze specifieker gemaakt worden zodat duidelijk is waar precies voor gewaarschuwd wordt. Zo niet, waar staat de terug te geven tekst gedefinieerd?

Reply:
Nee, dit is niet de inhoud van de warning. Dit is een omschrijving van de betekenis van de header

Reply:
Ok, dit regel je dus niet in het OAS3 bestand. Is dat omdat dat niet kan of omdat we een andere oplossing kiezen. Je zou zeggen dat die warning steeds hetzelfde moet zijn en dat je dat in het OAS3 bestand zou moeten kunnen regelen.

Reply:
Ik snap niet waar de verwarring zit, de OAS specifieert _schema's_, niet inhoud. Zo staat er voor een HTTP 201 response dat er een `Location` header in de response headers zit, maar de waarde daarvan specifieer je niet, omdat dat dynamisch is.

De waarde van `Warning` is ook dynamisch, dat is een vergelijkbare opzet. De OAS specifieert hier dus dat je response headers van de naam `Warning` kan verwachten, en die is ingericht volgens KP-API strategie.

Reply:
De vraag was alleen of het een soort enum betreft ipv een vrije tekst.

Antwoord: Het is een vrije tekst (maar wel volgens bepaalde richtlijnen)